### PR TITLE
Code Changes

### PR DESCRIPTION
**Pull request by aazeli**
- added vrmaifest file to run with steamvr
- option to change ip and port in config
- print and log options in config

### CODE REVIEW

**Comment by aazeli** [comment]
updated files are in 
https://github.com/suzedash/vrc-headpat-counter/releases/tag/v1.0.2

File: osc_counter/oscserver.py [hunk 1]
```diff
@@ -4,31 +4,49 @@
...
+        
+    def setConsoleTitle(self): 
+        ctypes.windll.kernel32.SetConsoleTitleW("VRCHeadpatCounter")
 
     # Entry point from OSC Unity receiver for any contact point
     # Remember from the README that args is derived from your avatars OSC JSON file
     # Per the README these address can only be boolean
     def _def_osc_dispatch(self, address: str, *args) -> None:
         if (self.messenger.timer.enough_time_passed() and \
-                self.messenger.has_new_content(tracker := self.filemanager.get_tracker())):
```
Comment (LincolnVR) [inline osc_counter/oscserver.py]: Both the indentation you have done here and the one I previously had are valid and shown in the PEP8 style guide.

As personal preference I would like to maintain having the additional indentation as it helps me quickly differentiate between the long if statement and the rest of the code block.

**Review by LincolnVR — CHANGES_REQUESTED** [review]
Just one change on the indentation on the if statement. 
Is this PR supposed to have an exe attached to it? I see that you changed the instructions on the documentation to run an exe, but I am not seeing it attached to this PR? 

Other than that I really love these additions and changes. Thanks for implementing version control as well.

**Comment by aazeli** [comment]
[VRCHeadpatCounter.zip](https://github.com/LincolnVR/vrc-headpat-counter/files/10323016/VRCHeadpatCounter.zip)

**Review by LincolnVR — APPROVED** [review]
Looks great! love the additions and added quality of life this brings

**Pull request merged**